README.md

```diff
@@ -56,55 +39,87 @@ for how to migrate from deeplearn.js to TensorFlow.js.
 
 ## Getting started
 
-Let's add a scalar value to a vector. TensorFlow.js supports _broadcasting_
-the value of scalar over all the elements in the tensor.
+There are two main ways to get TensorFlow.js in your JavaScript project:
+via <a href="https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_JavaScript_within_a_webpage" target="_blank">script tags</a> <strong>or</strong> by installing it from <a href="https://www.npmjs.com/" target="_blank">NPM</a>
+and using a build tool like <a href="https://parceljs.org/" target="_blank">Parcel</a>,
```

Comment:
Just curious is Browserify is now disfavored?

Reply:
I don't think so. I didn't include it because i find it slightly harder to configure (separation of concerns between browserify, babelify, watchify etc).